hooks/backend.py

```diff
@@ -140,7 +140,12 @@ def install(self, backend):
                 config.get('ssl-key-contents'))
 
     def start(self, backend):
-        pass
+        config = backend.config
+        utils.start_builtin_server(
+            config['ssl-cert-path'], config['serve-tests'],
+            config['sandbox'], config['builtin-server-logging'],
+            not config['secure'], config['charmworld-url'],
+            port=config.get('port'))
```

Comment:
I'm not sure what you mean--I'm just bringing back all of the available juju config options for the builtin server.